docs/getting_started_on_linux.md

```diff
@@ -68,6 +68,42 @@ environment variable is set when you are building. You may find it useful to add
 `source [PATH TO VULKAN SDK]/setup-env.sh` to your `~/.bashrc` file to simplify
 environment variable setup.
 
+## Optional: Configure Git
+
+### Git SSH
```

Comment:
I would just link the GitHub docs on this. There are so many key configuration options: https://help.github.com/en/enterprise/2.15/user/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent

Oh I see this mirrors the windows instructions

Reply:
Yeah. I just copy pasted it